sympy/plotting/plot.py

```diff
@@ -237,6 +237,16 @@ def append(self, arg):
         Plot object containing:
         [0]: cartesian line: x**2 for x over (-10.0, 10.0)
         [1]: cartesian line: x for x over (-10.0, 10.0)
+	
+	    .. plot::
+
+	       >>> from sympy import symbols
```

Comment:
incorrect indentation

Reply:
this is the formatting with which I got a proper output. With any other formatting, I was getting an error. If you could tell me the correct indentation, that'd be helpful.

Reply:
all indentation of the `>>>` lines should be the same

Reply:
Alright, I've corrected the indentation. What shall I do about the redundant code?